WalletWasabi.Gui/Controls/WalletExplorer/CoinJoinTabViewModel.cs

```diff
@@ -466,6 +472,11 @@ private TargetPrivacy TargetPrivacy
 			set => this.RaiseAndSetIfChanged(ref _targetPrivacy, value);
 		}
 
+		public bool IsLurkingWifeMode
+		{
+			get => Global.UiConfig.LurkingWifeMode == true;
+		}
```

Comment:
Does `public bool IsLurkingWifeMode => Global.UiConfig.LurkingWifeMode is true;` work?

Reply:
Yes!

Reply:
Yes!